.travis.yml

```diff
@@ -2,8 +2,11 @@ sudo: false
 language: ruby
 rvm:
   - 2.3.3
+  - 2.3.6
```

Comment:
Not sure if 2.3.3 should be tested separately.
I believe it was just latest 2.3 version at the moment

Reply:
Fixed